src/workflows-app/WorkflowsInWorkspace.ts

```diff
@@ -154,13 +168,44 @@ export const WorkflowsInWorkspace = ({
                           ),
                         ]
                       ),
+                      div(
+                        {
+                          style: {
+                            display: 'flex',
+                            flexDirection: 'column',
+                            marginTop: '30%',
+                          },
+                        },
+                        [
+                          h(
+                            MenuButton,
+                            {
+                              onClick: () => setMethodToDelete(method),
+                              disabled: !canWrite(accessLevel),
+                              tooltip: !canWrite(accessLevel)
+                                ? 'You must have write permission to delete workflows in this workspace'
+                                : '',
+                              style: {
+                                justifyContent: 'flex-end',
+                              },
+                            },
+                            [makeMenuIcon('trash'), 'Delete']
```

Comment:
I think we may want to make this icon a `cardMenuIcon` with a `MenuButton` giving the delete option and functionality. Reason being we'll be able to add more options to this in the future, like choosing a configuration for a workflow, etc. What do you think?

Reply:
(discussed in standup but for the record)

we decided not to use the `cardMenuIcon` because "delete" is the only available action, and didn't feel it was worth adding the extra click unnecessarily